src/features/machine/machineSlice.js

```diff
@@ -56,6 +57,10 @@ const machineSlice = createSlice({
       }
       state.rectOrigin = newValue
     },
+    toggleMinimizeMoves(state, action) {
+      state.minimizeMoves = !state.minimizeMoves
+      localStorage.setItem('minimizeMoves', state.minimizeMoves)
```

Comment:
From the user perspective, does minimize moves make more sense as a toggle in the output dialog? Similar to the reverse toggle?

Reply:
I don't think so, because you really need to see what it looks like before you output it. I've generally found that any pattern that has overlapping lines will not look good minimized because you clobber your elegant pattern when you draw over it out of sequence. Very occasionally, it actually _adds_ distance. I'm sure that could be fixed, but I concluded for now anyway that it wasn't worth changing.